index.html

```diff
@@ -227,6 +227,7 @@ <h4 class="popup__type">Квартира</h4>
       <button type="button" class="error__button">Попробовать снова</button>
     </div>
   </template>
-  <script src='js/main.js'></script>
+  <script src="../node_modules/lodash/lodash.min.js"></script>
```

Comment:
Тут адрес неправильный, нужно
```suggestion
  <script src="./node_modules/lodash/lodash.min.js"></script>
```

Reply:
а почему с одной точкой?
Я же должен из данной папки подняться на уровень выше, всегда же использовалось для этого: ".."

Reply:
Посмотри внимательно. У тебя `index.html` и папка `node_modules` на одном уровне